src/main/kotlin/build/buf/intellij/resolve/BufCacheLibrary.kt

```diff
@@ -19,11 +19,14 @@ import build.buf.intellij.icons.BufIcons
 import com.intellij.navigation.ItemPresentation
 import com.intellij.openapi.roots.SyntheticLibrary
 import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.openapi.vfs.VirtualFileManager
+import java.nio.file.Path
+import java.util.*
 import javax.swing.Icon
 
-class BufCacheLibrary(private val bufCacheFolder: VirtualFile) : SyntheticLibrary(), ItemPresentation {
+class BufCacheLibrary(private val name: String, private val srcRoots: Collection<String>) : SyntheticLibrary(), ItemPresentation {
     override fun getPresentableText(): String {
-        return BufBundle.message("syntactic.library.text")
+        return BufBundle.message("syntactic.library.text", name)
```

Comment:
The IntelliJ plugin at some point needs to become module aware - this will at least add multiple modules with decorations on them to indicate which module they came from.

Here's an example:
<img width="352" alt="image" src="https://github.com/bufbuild/intellij-buf/assets/398111/f5a1f0ea-76f4-4862-b511-6cf0e95298d5">